src/modules/core/components/Warning/Warning.css

```diff
@@ -0,0 +1,14 @@
+.warning {
+  display: flex;
+  flex-direction: column;
+  padding: 25px 34px;
```

Comment:
I think this padding should be `12px` , and then the text inside the container should have its own padding, so something like `padding: 13px 22px 0 22px`. 

This is because I noticed that in the design the button doesn't have the same padding as the text